src/checkers/cvechecker.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (C) 2018 Endless Mobile, Inc.
+#
+# Authors:
+#       Andrew Hayzen <ahayzen@gmail.com>
+#       Joaquim Rocha <jrocha@endlessm.com>
```

Comment:
(I don't mind but) you don't have to add my name if I didn't touch this code or if most of it wasn't copied from my code.